packages/create-toolpad-app/src/templates/dashboardLayout.ts

```diff
@@ -0,0 +1,12 @@
+export default `import * as React from 'react';
```

Comment:
I could see us make each of those into a common shape like:

```tsx
interface File {
  content: string
}

type Template = (option: Options) => File
```

That way a template has input, and can also return metadata beyond content